lattices/LatticeMath/StatsTiledCollapser.tcc

```diff
@@ -47,7 +49,7 @@ template <class T, class U>
 void StatsTiledCollapser<T,U>::initAccumulator (uInt n1, uInt n3) {
    _sum = new Block<U>(n1*n3);
    _sumSq = new Block<U>(n1*n3);
-   _npts = new Block<U>(n1*n3);
+   _npts = new Block<Double>(n1*n3);
```

Comment:
are n1 and n3 data dimensions? if so care should be taken to not overflow the int, e.g. by using int64